torch/_dynamo/variables/builder.py

```diff
@@ -436,9 +437,10 @@ def index_source(key):
                     return key
 
             result = {
-                k: VariableBuilder(
-                    self.tx, GetItemSource(self.get_source(), index_source(k))
-                )(value[k]).add_guards(guards)
+                k: LazyVariableTracker(
```

Comment:
Feels like export should always just realize here too?

Reply:
I think the one in symbolic_convert.py should cover it.